rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.75"
```

Comment:
What's up with that change?

Reply:
I cannot compile this repo without changing 1.75 to 1.75.0 and I have no idea about it. I will change it back to 1.75 now.

Reply:
Would you mind opening an issue with the logs included? I'll take a look at it.

Reply:
Sure

Reply:
@osiewicz Issue opened: https://github.com/zed-industries/zed/issues/6925